docs/resources/dataplatform_node_pool.md

```diff
@@ -27,7 +27,7 @@ resource "ionoscloud_dataplatform_cluster" "example" {
     day_of_the_week  	= "Sunday"
     time				= "09:00:00"
   }
-  version	= "23.7"
+  version	= "24.11"
```

Comment:
you could think of adding a comment to tell users to use `ionosctl dataplatform version active` which  returns the latest version